### PR TITLE
step-registry: add `e2e-openstack-csi` workflow

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

**Comment by adduarte** [comment]
This looks good to me. 
All that is left to create a job is to 
add a reference to the workflow somewhere in ci-operator/config and run 
"make jobs" to generate the new job file to submit.

File: ci-operator/step-registry/openshift/e2e/openstack/csi/openshift-e2e-openstack-csi-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    post:
+    - chain: ipi-openstack-post
+    env:
+      OPENSTACK_EXTERNAL_NETWORK: "external"
+      OS_CLOUD: "openstack-cloud"
+      OPENSTACK_COMPUTE_FLAVOR: "m1.s2.xlarge"
+      CONFIG_API_VERSION: "v1beta4"
+      CONFIG_TYPE: "minimal"
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      TEST_INSTALL_CSI_DRIVERS: openstack-cinder
```
Comment (jsafrane) [inline ci-operator/step-registry/openshift/e2e/openstack/csi/openshift-e2e-openstack-csi-workflow.yaml:17]: What does env. variable supposed to do? `TEST_INSTALL_CSI_DRIVERS` can install only very old version of upstream AWS CSI driver, I should remove it.

For OCP AWS CSI driver tests, we use this workflow:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/aws/csi/openshift-e2e-aws-csi-workflow.yaml

It's just a common aws installation workflow, only with this extra step `storage-conf-csi-aws-ebs`:
https://github.com/openshift/release/tree/master/ci-operator/step-registry/storage/conf/csi-aws-ebs
This step copies manifest.yaml from `aws-ebs-csi-driver-operator-test` image to `$SHARED_DIR`.

And finally, `aws-ebs-csi-driver-operator-test` image is created like this:
https://github.com/openshift/aws-ebs-csi-driver-operator/blob/master/Dockerfile.test
(i.e. just copy of the last forked repo, with the test manifest).

**Pull request merged**